src/compiler/types.ts

```diff
@@ -451,8 +451,6 @@ export const enum SyntaxKind {
     NotEmittedStatement,
     PartiallyEmittedExpression,
     CommaListExpression,
-    MergeDeclarationMarker,
-    EndOfDeclarationMarker,

```

Comment:
Should we leave these in but toss a `@deprecated` and maybe a comment saying they're unused, at least until such a time were we direly need to reclaim the numbers? It might just be a lil' odd if we reuse these syntax kind numbers in the future. It'd reduce the API changes to zero, anyway.

Reply:
Is that necessary? The types they represented were `/** @internal */`, and the number values for `SyntaxKind` change often.